File: sololearn/GothamCity/GothamCity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int criminals = Convert.ToInt32(Console.ReadLine());
            
            if (criminals < 5)
                Console.WriteLine("I got this!"); 
            else if (criminals < 10)
                Console.WriteLine("Help me Batman"); 
            else
                Console.WriteLine("Good Luck out there!"); 
        }
    }
}

File: sololearn/GothamCity/GothamCity.swift
import Foundation

if let typed = readLine() {
    if let criminals = Int(typed) {
        if criminals < 5 {
            print("I got this!")
        } else if criminals < 10 {
            print("Help me Batman")
        } else {
            print("Good Luck out there!")
        }
    }
}
